examples/with-solana/README.md

```diff
@@ -166,7 +166,138 @@ Token balance for warchest: 0.0001
+2. Attempt Transfer - since we're attempting the transfer WITHOUT having created the policy allowing it, this will fail
+3. Create Token Policy - this will create a policy that allows Solana transactions initiated by the correct user to be signed if they contain a single instruction which is an SPL token transfer to the correct token account
+4. Attempt Transfer (again) - this will succeed, now that the appropriate policy has been created!
+
+NOTE: A very key piece of understanding to glean from this example is the use of "associated token addresses (or "associated token account address") in the creation of SPL related policies. For further context on ATA's look here: https://spl.solana.com/associated-token-account
+
+#### Step 1. Setup
+
+First we will run the setup command
+
+```bash
+$ pnpm token-transfer-policy setup
+```
+
+This will create a Solana wallet address that you'll have to fund with devnet SOL via a faucet or cli. A commonly used SOL devnet faucet is available here --> https://faucet.solana.com/
+
+Funding your devnet and hitting `y` in the terminal to proceed will set up the necessary on-chain state and Turnkey setup to start, and will output the following information to be used in coming steps
+
+- Turnkey Solana wallet address
+- Token Mint public key
+- Non root user created with user id
+
+```bash
+✔ Ready to Continue? … yes
+Broadcasting token creation transaction...
+Transaction broadcast and confirmed! 🎉
+	https://explorer.solana.com/tx/35SJqgdy2nWBvuECXrqRVkuYdRf6kkmWCuxNFd3TcmdMsXfrEns6nLjQqhKx5zvUwYPZ8xHnsiYCRtuhS8a9a5XX?cluster=devnet
+
+Broadcasting token account creation transaction...
+Transaction broadcast and confirmed! 🎉
+	https://explorer.solana.com/tx/3ffN8Goe4P3TSbJZGrmBADfGZWVdnrHNTsbNqce8AAPbANFmhWVHhd5MMhinXhppccMMvjqPqj8vz1PbFQUaV8Ns?cluster=devnet
+
+Broadcasting token account creation transaction...
+Transaction broadcast and confirmed! 🎉
+	https://explorer.solana.com/tx/4sTgMDSMvf1oTJenJWiGRapf6mhoghv85juqdTv5VmZuyQ48ykgf22RZPxdCACSEEg8ige6GarSEQc2Nr4NkfzY1?cluster=devnet
+
+Broadcasting mint transaction...
+Transaction broadcast and confirmed! 🎉
+	https://explorer.solana.com/tx/FMBBXrXYGb2nGQqbHs1j17GJG5owzSMnZ1RqN9R7cSVn8Q2JMbZKMik9MLzvG1LNmdrsmSV16yvFdyLHukZw5KJ?cluster=devnet
+
+New user created!
+- Name: Non Root User
+- User ID: 13c4609b-8818-40be-aeac-c8b63da0de4a
+
+Setup complete -- token mint and token accounts created, non root user created
+Turnkey Solana wallet address: 2bwUkGZ72SNyzBWFwfzKVrVWJ9JNZggjqFPDM9X9mrp9
+Token Mint public key: FA7umztm6eYcENE22ndEZhd8MCf9C7myJ2KjfCwobj8F
+Non root user created with user id: 13c4609b-8818-40be-aeac-c8b63da0de4a
+```
+
+#### Step 2. Attempt to Transfer Tokens (This is expected to FAIL)
+
+We will now attempt to transfer these SPL tokens that we've created
+
+```bash
+$ pnpm token-transfer-policy attempt_transfer
```

Comment:
tiny nit: wanna use kebab case here `attempt_transfer` for consistency?

Reply:
yessir, done